lib_vehicle_model/include/lib_vehicle_model/VehicleModelControlInput.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+/*
+ * Copyright (C) 2018-2019 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+namespace lib_vehicle_model {
+  /**
+   * @struct VehicleModelControlInput
+   * @brief A struct used to define the set of variables used to represent the vehicle control input.
+   * 
+   * The members of this struct are defined relative to the vehicle state and control models described in the VehicleModelLib documentation
+   */
+  struct VehicleModelControlInput 
```

Comment:
I have very similar design concerns when designing maneuvers on what suitable values this control input class should contain. There are some many variations. The Autoware outputs are desired velocity and desired curvature. Their new pacmod_interface outputs are desired throttle pedal and desired steering wheel angle. See https://github.com/CPFL/Autoware/issues/1944 for detail. I think we need to consider what data we should support more carefully to make consumers to use this package in a easier way. I totally agree that acceleration and steering_angle is the common set of commands.

Reply:
I very much agree. What I have above is a good start, but there is still a disconnect with maneuvers and Autoware.

Reply:
Yes. We need to address it later. It should not be too hard to change the variable name later.